package.json

```diff
@@ -92,5 +92,8 @@
       "**/node_modules/**",
       "**/build/"
     ]
+  },
+  "devDependencies": {
```

Comment:
Now we have all kind of deps in "dependencies". It makes sense to split them I suppose since now we have "normal" dependencies in root package.json . What do you think?